library/headers.inc.php

```diff
@@ -102,6 +129,4 @@ function resolveFancyboxCompatibility(){ ?>
 <?php 
 } 
 //The following seem to always get called.  I don't believe this is proper.
-?>
-<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/dialog.js"></script>
+?>
```

Comment:
Newline please.
Personally, I like if(){  vs if() {
-Not really clean to me, but it is fine.

Reply:
I prefer if() {

I think this is the kind of stuff we get used to that our mind will never change

Reply:
I prefer if ()
             {

             } else {

             }

Reply:
:(

Reply:
:)